### PR TITLE
feat(hardhat-helpers): standalone contracts deploy functions

### PR DESCRIPTION
**Pull request by clemsos**
# Description

This refactoring adds some helpers function for contract deployments so it is easier to deploy on zk chains (see #13078 )

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: packages/hardhat-helpers/src/deploy.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+export const deployContract = async (
```
Comment (julien51) [inline packages/hardhat-helpers/src/deploy.js:1]: Do we also need a `deployUpgradableContract` helper?
Maybe in a follow up PR?

**Review by julien51 — COMMENTED** [review]
Did you replace in all the scripts to deploy contracts in the `governance` folder?

**Comment by julien51** [comment]
Do we have tests for `governance`? Can we just add a file that runs in CI that deploys all contracts on a "local" node so we can make sure the scripts are working and we don't end up finding that they don't work when we need to deploy a new network/contract?

File: packages/hardhat-helpers/src/deploy.js (hunk 1 above)
Comment (clemsos) [reply packages/hardhat-helpers/src/deploy.js:1]: yes, added too here

**Comment by clemsos** [comment]
> Do we have tests for `governance`? Can we just add a file that runs in CI that deploys all contracts on a "local" node so we can make sure the scripts are working and we don't end up finding that they don't work when we need to deploy a new network/contract?

That will be good indeed. We have tests already in `governance` folder but they are js unit tests. Will be good to have a way to test the cli tasks

File: governance/scripts/deployments/uniswap-v2.js
Comment (clemsos) [inline governance/scripts/deployments/uniswap-v2.js:1]: I removed all tasks and scripts related to Uniswap V2 deployment

**Review by julien51 — APPROVED** [review]
LGTM!

**Comment by julien51** [comment]
> That will be good indeed. We have tests already in governance folder but they are js unit tests. Will be good to have a way to test the cli tasks

Want to add that in another PR maybe?

**Comment by clemsos** [comment]
> > That will be good indeed. We have tests already in governance folder but they are js unit tests. Will be good to have a way to test the cli tasks
> 
> 
> 
> Want to add that in another PR maybe?

Yes I added a basic one here but can add more coverage in a separate PR

**Pull request merged**